### PR TITLE
Progress bar update

### PR DESCRIPTION
**Pull request by OlaShabalina**
## Overview of the Pull Request:
* Small task to download the progress bar icons and update them in the folder (based on how many dots are selected).
* Using the original pattern `q1-progress-dots.svg` with the number equal to the number of dots.

## link to Trello card or Github issue
[click here](https://trello.com/c/fC0gk1Gg)

## How Has This Been Tested?
- Please describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce. 
- If necessary, please also list any relevant details for your test setup/configuration

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [x] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [x] Is your pull request up-to-date with `main` branch?
- [x] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [x] Have you written instructions on how to test that your changes work as expected?
- [x] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

**Comment by OlaShabalina** [comment]
I like the original format `q1-progress-dots.svg`, regardless how the pages are names - the dots are showing the order

File: pages/form/Suggestions.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import { useState } from "react";
+import { Box, Heading, Text, Textarea } from "@chakra-ui/react";
+import Layout from "../../components/Layout/Layout";
+import useForm from "../../components/FormProvider";
+import {
+  BackButton,
+  ContinueButton
+} from "../../components/LinkButton/LinkButton";
+import Q7Progress from "../../public/images/progress-bar/q7-progress-dots.svg";
```
Comment (Amutha37) [inline pages/form/Suggestions.jsx:9]: q7-progress-dots.svg is not in the progress bar file yet. Is this something you have done locally.

**Review by Amutha37 — COMMENTED** [review]
About the progress bar that you mention old design. We just need to confirm with the design team.

File: pages/form/Suggestions.jsx (hunk 1 above)
Comment (OlaShabalina) [reply pages/form/Suggestions.jsx]: Hi @Amutha37 , ops, this PR is for progress-bar icons only, apologies. I replaced the icons and kept original name.
Goodpickup!

Comment (OlaShabalina) [reply pages/form/Suggestions.jsx]: Link to the Suggestions.jsx PR - (https://github.com/CodeforAustralia/council-emissions-calculator/pull/93)

**Review by batteries76 — APPROVED** [review]

File: pages/form/Suggestions.jsx (hunk 1 above)
Comment (Amutha37) [reply pages/form/Suggestions.jsx]: This is all good.

**Review by Amutha37 — APPROVED** [review]

**Pull request merged**